{
  logger.error("RetryOperationCallback call was interrupted for account {}: {}",directPaymentStateContext.getAccount().getExternalKey(),e.getMessage());
  return new OperationException(e,getOperationResultOnException(directPaymentStateContext));
}
